docs/DeveloperGuide.md

```diff
@@ -313,6 +313,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Exit the program**
+
+**MSS**
+
+1.  User requests to exit the program
+2.  System saves current data and program ends
```

Comment:
Maybe the use case should only describe the externally visible behavior, not internal details, of a system?

Reply:
Agreed, have made the change!